openshift/templates/patroni.yaml

```diff
@@ -152,6 +152,20 @@ objects:
                   value: 0.0.0.0:5432
                 - name: PATRONI_RESTAPI_LISTEN
                   value: 0.0.0.0:8008
+                - name: ROCKET_AUTH_TOKEN
+                  valueFrom:
+                    secretKeyRef:
+                      name: ${NAME}-global
+                      key: rocket.chat-auth-token-secret
+                - name: ROCKET_USER_ID
+                  valueFrom:
+                    secretKeyRef:
+                      name: ${NAME}-global
+                      key: rocket.chat-user-id-secret
```

Comment:
should user IDs be kept secret? because I just moved the user id for bcfw p1 authentication from a secret to a config

Reply:
I noticed that - but didn't want to have more to and fro since I think this borders on the philosophical.  I consider a username secret since it's part of a login/identity, so I would prefer it be a secret. Two parts to hacking right - you need a username and a password.

Reply:
yeah now that I think about it more, it seems safer to me to include user IDs in secrets. From what I understand there's no harm in having a few extra variables in Secrets rather than ConfigMaps, and it should add a bit of extra security